pages/profiles/MadagascarOLE.md

```diff
@@ -0,0 +1,15 @@
+# Hello All !!!!
+---
```

Comment:
By using Heading, you will get a free horizontal rule (check out [here](http://open-learning-exchange.github.io/#!pages/githubandmarkdown.md#Prerequisites)) and that's the reason why there are two horizontal rules on your [rawgit](https://rawgit.com/MadagascarOLE/MadagascarOLE.github.io/Mada-ole-md-wiki/#!pages/profiles/MadagascarOLE.md). Please delete line 2. Other than that, it looks good  to me 👍